doc/api/fs.md

```diff
@@ -1447,6 +1453,9 @@ The listener callback gets two arguments `(event, filename)`.  `event` is either
 `'rename'` or `'change'`, and `filename` is the name of the file which triggered
 the event.
 
+Please note the listener callback is attached to the `'change'` event
```

Comment:
@thefourtheye I could change the name in both the code and docs to `eventType` or something like that - it's just a change of identifier name so it wouldn't be intrusive/breaking